nemo/collections/nlp/modules/common/huggingface/huggingface_utils.py

```diff
@@ -90,7 +91,7 @@ def get_huggingface_lm_model(
 
     if model_type in HUGGINGFACE_MODELS:
         model_class = HUGGINGFACE_MODELS[model_type]["class"]
-        if config_file:
+        if config_file and os.path.exists(config_file):
```

Comment:
Raise an error as use has specified a config_file but it does not exists?

Reply:
well, that will make at least some pre-trained models unusable. specifically those which we have on NGC right now :D

Reply:
I think having a warning is the minimum we should have. We are skipping reading a file without even notifying the user. 
Those checkpoints on the NGC, have they used this config_file? If yes, how are they going to work without it?

Reply:
can't do that - it will make some pre-trained models unusable. Specifically, the ones from NGC :)